### PR TITLE
Ea definition support

### PR DESCRIPTION
**Pull request by JkhatriInfobox**

### CODE REVIEW

File: tests/unit/plugins/modules/test_nios_a_record.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def _get_wapi(self, test_object):
         wapi = api.WapiModule(self.module)
         wapi.get_object = Mock(name='get_object', return_value=test_object)
         wapi.create_object = Mock(name='create_object')
-        wapi.update_object = Mock(name='update_object')
+        wapi.update_object = Mock(name='update_object', )
```
Comment (JchhatbarInfoblox) [inline tests/unit/plugins/modules/test_nios_a_record.py:56]: remove extra comma.

**Review by JchhatbarInfoblox — CHANGES_REQUESTED** [review]
Update requested changes

File: tests/unit/plugins/modules/test_nios_a_record.py (hunk 1 above)
Comment (JkhatriInfobox) [reply tests/unit/plugins/modules/test_nios_a_record.py]: Fixed

**Review by JchhatbarInfoblox — APPROVED** [review]
LGTM

**Pull request merged**